lib/logstash/config/cpu_core_strategy.rb

```diff
@@ -0,0 +1,29 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "logstash/config/defaults"
+
+module LogStash module Config module CpuCoreStrategy
```

Comment:
is this our new indenting style? I've been making christmas trees. Has this been discussed?